projects/court-case-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/proxy/CommunityApiController.kt

```diff
@@ -343,7 +352,7 @@ class CommunityApiController(
         return proxy(request)
     }
 
-    fun proxy(request: HttpServletRequest): Any {
+    fun proxy(request: HttpServletRequest): ResponseEntity<ByteArray> {
```

Comment:
Will this load the whole doc into memory as a byte array?  Could cause crashes when downloading large files?

Reply:
hmm you are right it will. Ill change it so that the call to community api uses a Stream